database/migrations/2022_02_19_161458_create_inspects_table.php

```diff
@@ -21,6 +21,7 @@ public function up()
             $table->date('date')->nullable();
             $table->string('name')->nullable();
             $table->string('company')->nullable();
+            $table->string('engineer')->nullable();
```

Comment:
question : since we store engineer name inside inspects table, what happen if that engineer change its name? for example during this inpsection form submission, my name in database is "Imran", after submit, few weeks later I change my profile (as an engineer) to name "Muhammad" ?